cmd/execute-cadence-script/README.md

```diff
@@ -0,0 +1,51 @@
+# Execute Cadence Script
+
+## Description
+
+This utility binary can be used to execute a Cadence script against a running Flow network. By default, access nodes are expected on localhost:3569. Script can be executed on an arbitrary block height.
+
+## Usage
+
+```sh
+Usage of ./execute-cadence-script:
+  -a, --api string           access node API address (default "localhost:3569")
+  -h, --height int           height on which to execute script (default -1)
```

Comment:
Most users will probably be confused with a negative value for height. I recommend switching to a pointer value and just leave it nil when nothing is specified. Default can then be shown as `(default: last indexed height)` or something along those lines, which is much clearer than `-1` IMO. WDYT?

Reply:
That's a good idea, but it looks like `pflag` does not really expect such a use case, everything I looked into would look too awkward and clunky :/ I've improved the help message to better reflect what the default value indicates.

That, plus `-1` value is often used as a sentinel, I believe it's clear enough..

Reply:
`pflag` actually returns a `*uint64` already, so it's actually perfectly handling this case 😄 

```go
var flagHeight *uint64

pflag.Int64VarP(flagHeight, "height", "h", nil, "height on which to execute script")

height = state.LastHeight() // pseudo-code
if flagHeight != nil {
	height = *flagHeight
}
```

Or am I missing smth?

Reply:
No, the first argument expects an address of an existing variable to write to (will dereference a nil pointer there), and the fourth argument can't be nil as it expects an `int64`/`uint64` (depending on the function).

Reply:
You're absolutely right! I really don't like the UX though but this is something we can think about later, if and when we switch from `spf13/pflag` to something that allows us to have optional parameters with no default value.